### ISSUE TITLE
Cannot access API using cloudflare workers

### ISSUE THREAD

**Issue by pedrohgmacedo** [google-gemini/generative-ai-js#37]
## Expected Behavior
I tried to create a simple Telegram chatbot for personal use, using cloudflare workers. All their [IP Ranges](https://www.cloudflare.com/ips/) are from an [Available Region](https://ai.google.dev/available_regions), and so is my acount where I generated the API Key. My use case falls strictly within the [Terms](https://ai.google.dev/terms)

## Actual Behavior
 Error: [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1/models/gemini-pro:generateContent: [400 Bad Request] User location is not supported for the API use.

## Steps to Reproduce the Problem

1. Run this code in a (deployed) cloudflare worker (:
```ts
const genAI = new GoogleGenerativeAI(GEMINI_API_KEY);
const model = genAI.getGenerativeModel({ model: "gemini-pro"});
const prompt = "Write a story about a magic backpack."

const result = await model.generateContent(prompt);
const response = await result.response;
const text = response.text();
```
It always returns a 400 Bad Request error.

**Comment by jfreeg** [comment]
Unfortunately, I am encountering the same problem. This is because of regulatory issues in the EU. I hope it is resolved soon.

**Comment by jfreeg** [comment]
Try using Gemini with the vertex.

**Comment by hsubox76** [comment]
I think available regions may have expanded lately, is anyone still having this problem?

**Comment by pedrohgmacedo** [comment]
> I think available regions may have expanded lately, is anyone still having this problem?

I don't know, I'm not using Gemini anymore.

**Comment by marb2000** [comment]
Pedro, I would love to heard the reasons of why Gemini is not a valid solution for it. How can we improve it?

**Comment by pedrohgmacedo** [comment]
> Pedro, I would love to heard the reasons of why Gemini is not a valid solution for it. How can we improve it?

Frankly, this issue was sitting here for about 4 months. I simply moved on.   
Technical issues like the one above aside, in my brief experiments with Gemini on the gemini.google.com website as well as on the AI Studio, I have found that the model is prone to hallucinate towards political bias when queried for facts, to the point of falsely stating simple uncontroversial facts (i.e. information that I could find in wikipedia), in a way that no other model, open source or private, does with the same exact prompt.  
One way that google's models could improve is to be more factual.  
Another personal use I have to to make memes to share among friends. Currently only open source can be used for this.  
I feel uncomfortable using Gemini for anything that requires accurate or factual answers giving my anecdotal impressions.
